### ISSUE TITLE
cam release is incrementing minor number erroneously 

### ISSUE THREAD

**Issue by bess** [yalelibrary/YUL-DC#489]
I just released blacklight, and I expected it to release v1.13.1 b/c there was no `Feature` label on any of the merged PRs. However, it tagged v1.14.0 instead.

**Comment by bess** [comment]
It appears from the Github GUI that there should be two PRs merged since blacklight v1.13.0, neither of which has a `Feature` label. However, when I query via the github API I see three PRs:

```
(byebug) pull_requests.map { |a| puts a[:merged_at] if a[:merged_at] && a[:merged_at] > last_release_timestamp }
2020-08-25 12:09:34 UTC
2020-08-24 17:55:32 UTC
2020-08-20 21:14:26 UTC
```

And when I compare the timestamp on that last PR to the timestamp of our last release, I see that it was merged one second after the last release:

```
(byebug) last_release_timestamp
2020-08-20 21:14:25 UTC
```

**Comment by martinlovell** [comment]
Some info that might be related:  
For the v2.11.6 release of the management app, one PR: #245 had the Feature label. 
Deploying only incremented the minor version. (Last release was v2.11.5).
We added the Feature label just before doing the release.
